docs/en_US/NAS/QuickStart.rst

```diff
@@ -0,0 +1,198 @@
+Quick Start of Retiarii on NNI
+==============================
+
```

Comment:
another thought is to merge the one-shot approach into 1 quick start which includes:
- define model space
- [A] directly use one-shot trainer
- [B] go with muliti-trial mode: (1) define search strategy; (2) define evaluator;
- start experiment
- view experiment results